docs/developer/available-plugins/dummy-credit-card.mdx

```diff
@@ -80,21 +80,20 @@ Completing checkout and testing gateway response:
 ```graphql {2-4}
 mutation {
   checkoutComplete(
-      checkoutId: "Q2hlY2tvdXQ6YjIyN2JiNWYtODA4Ny00ZjNlLThhMjYtMzE1OGI4NWE4N2Jh"
+    checkoutId: "Q2hlY2tvdXQ6YjIyN2JiNWYtODA4Ny00ZjNlLThhMjYtMzE1OGI4NWE4N2Jh"
```

Comment:
Since some time we recommend using `token` as identifier in checkout - @CossackDex could you update this example and the one above with `checkoutPaymentCreate` to use `token` instead of `id`?